js/supermarkets.js

```diff
@@ -47,7 +46,7 @@ function searchMarketsNear(searchedPlace) {
             showStoresMarker(data);
             setOnClickListener();
             autoCompleteInSearch()
-            triggerBtn();
+            triggerFunction();
```

Comment:
what is expected behavior when this line gets executed? 
As far as I can see only addEventListener will be added for the input to be able to react on click, but this could be done without extra function.